app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
+    <string name="pref_gesture_action_double_click">Double click</string>
+    <string name="pref_gesture_action_move_pointer">Move pointer</string>
+    <string name="pref_gesture_action_pan">Pan local frame</string>
+    <string name="pref_gesture_action_remote_scroll">Scroll remote content</string>
+    <string name="pref_gesture_action_remote_drag">Drag remote content</string>
+    <string name="pref_gesture_action_open_keyboard">Open keyboard</string>
 
     <string name="pref_theme">Theme</string>
-    <string name="pref_theme_option_system">Follow System Theme</string>
+    <string name="pref_theme_option_system">System</string>
     <string name="pref_theme_option_dark">Dark</string>
     <string name="pref_theme_option_light">Light</string>
-    <string name="pref_prefer_advanced_editor">Prefer Advanced Editor</string>
-    <string name="pref_prefer_advanced_editor_summary">Enable this to directly open Advanced mode to add/edit Servers</string>
-    <string name="pref_sort_server_list">Sort Server List</string>
-    <string name="pref_sort_server_list_summary">Servers are sorted by their name</string>
+    <string name="pref_prefer_advanced_editor">Prefer advanced editor</string>
+    <string name="pref_prefer_advanced_editor_summary">Opens advanced mode when adding/editing servers</string>
+    <string name="pref_sort_server_list">Sorting of servers</string>
+    <string name="pref_sort_server_list_summary">By name</string>
     <string name="pref_gesture">Gestures</string>
-    <string name="pref_single_tap">Single Tap</string>
-    <string name="pref_two_finger_tap">2 Finger Tap</string>
+    <string name="pref_single_tap">Single tap</string>
+    <string name="pref_two_finger_tap">Two-finger Tap</string>
     <string name="pref_double_tap">Double Tap</string>
-    <string name="pref_long_press">Long Press</string>
-    <string name="pref_swipe1">1 Finger Swipe</string>
-    <string name="pref_swipe2">2 Finger Swipe</string>
-    <string name="pref_drag">Drag (Long Press then Swipe)</string>
+    <string name="pref_long_press">Long press</string>
+    <string name="pref_swipe1">1 Finger swipe</string>
+    <string name="pref_swipe2">2 Finger swipe</string>
+    <string name="pref_drag">Drag (long press then swipe)</string>
     <string name="pref_mouse">Mouse</string>
-    <string name="pref_mouse_passthrough">Mouse Passthrough</string>
-    <string name="pref_mouse_passthrough_summary_off">Mouse events are used locally for gestures</string>
-    <string name="pref_mouse_passthrough_summary_on">Mouse events are sent directly to Server</string>
-    <string name="pref_hide_local_cursor">Hide Local (Android) Cursor</string>
+    <string name="pref_mouse_passthrough">Mouse passthrough</string>
+    <string name="pref_mouse_passthrough_summary_off">Using mouse events for gestures on Android</string>
+    <string name="pref_mouse_passthrough_summary_on">Sending all mouse events to server</string>
```

Comment:
These two seems unnecessary to me. Original strings feel more 'stable', and 'continuing' nature of new strings feels odd (at-least to me).